vars/isUpstreamTrigger.txt

```diff
@@ -1,5 +1,6 @@
-Check if the build was triggered by an upstream job.
+Check if the build was triggered by an upstream job, being it possible to add some filters.
 
 ```
 def upstreamTrigger = isUpstreamTrigger()
+def upstreamTrigger = isUpstreamTrigger(filter: 'PR-')
```

Comment:
Missing the new argument and the description

Reply:
I added it and also improved the description, thanks!